tailwind.config.js

```diff
@@ -0,0 +1,186 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {
+      backgroundImage: {
+        unit8: "url('/assets/images/unit8(1).svg')",
```

Comment:
`unit8(1)` as an image name is not descriptive enough. As there is another image named `unit8` already adding `(1)` doesn't explain much about their differences.